src/scss/components/ec-btn/ec-btn.story.js

```diff
@@ -105,92 +119,127 @@ stories
-        <button class="ec-btn ec-btn--sm ec-btn--success ec-m--8">Success</button>
-        <button class="ec-btn ec-btn--sm ec-btn--error ec-m--8">Error</button>
-        <button class="ec-btn ec-btn--sm ec-m--8" disabled>Disabled</button>
-
-        <h3 class="ec-m--8">Solid Rounded Buttons</h3>
-        <button class="ec-btn ec-btn--md ec-btn--rounded ec-btn--primary ec-m--8">Primary</button>
-        <button class="ec-btn ec-btn--md ec-btn--rounded ec-btn--secondary ec-m--8">Secondary</button>
-        <button class="ec-btn ec-btn--sm ec-btn--rounded ec-btn--success ec-m--8">Success</button>
-        <button class="ec-btn ec-btn--sm ec-btn--rounded ec-btn--error ec-m--8">Error</button>
-        <button class="ec-btn ec-btn--sm ec-btn--rounded ec-m--8" disabled>Disabled</button>
-
-        <h3 class="ec-m--8">Outline Buttons</h3>
-        <button class="ec-btn ec-btn--md ec-btn--outline ec-btn--primary ec-m--8">Primary</button>
-        <button class="ec-btn ec-btn--md ec-btn--outline ec-btn--secondary ec-m--8">Secondary</button>
-        <button class="ec-btn ec-btn--sm ec-btn--outline ec-btn--success ec-m--8">Success</button>
-        <button class="ec-btn ec-btn--sm ec-btn--outline ec-btn--error ec-m--8">Error</button>
-        <button class="ec-btn ec-btn--sm ec-m--8 ec-btn--outline" disabled>Disabled</button>
+  .add('all buttons (dark)', generateAllForElement('button'), {
+    backgrounds: [darkTheme],
+  })
+  .add('all buttons (light)', generateAllForElement('button'), {
+    backgrounds: [lightTheme],
+  })
+  .add('all anchors (dark)', generateAllForElement('a'), {
+    backgrounds: [darkTheme],
+  })
+  .add('all anchors (light)', generateAllForElement('a'), {
+    backgrounds: [lightTheme],
+  });
 
-        <h3 class="ec-m--8">Outline Rounded Buttons</h3>
-        <button class="ec-btn ec-btn--md ec-btn--rounded ec-btn--outline ec-btn--primary ec-m--8">Primary</button>
-        <button class="ec-btn ec-btn--md ec-btn--rounded ec-btn--outline ec-btn--secondary ec-m--8">Secondary</button>
-        <button class="ec-btn ec-btn--sm ec-btn--rounded ec-btn--outline ec-btn--success ec-m--8">Success</button>
-        <button class="ec-btn ec-btn--sm ec-btn--rounded ec-btn--outline ec-btn--error ec-m--8">Error</button>
-        <button class="ec-btn ec-btn--sm ec-btn--rounded ec-btn--outline ec-m--8" disabled>Disabled</button>
-      </div>
-    `,
-  }))
-  .add('icon buttons', () => ({
+function generateAllForElement(element) {
+  return () => ({
     components: { EcIcon },
-    props: {
-      iconName: {
-        default: select('Icon name', ['simple-check', 'simple-add', 'simple-dashboard', 'simple-sign-out'], 'simple-add'),
-      },
-      iconSize: {
-        default: number('Icon size', 20),
-      },
-      buttonSize: {
-        default: select('Size', ['ec-btn--sm', 'ec-btn--md'], 'ec-btn--sm'),
-      },
+    data() {
+      return {
+        element,
+        buttons: [
```

Comment:
I don't see this data being used anywhere.

Reply:
Solved